web/src/app/users/UserContactMethodCreateDialog.js

```diff
@@ -22,16 +22,40 @@ export default function UserContactMethodCreateDialog(props) {
     name: '',
     type: 'SMS',
     value: '',
+    countryCode: 'US', // default
```

Comment:
Store these in their own state as they have nothing to do with the contact method